### PR TITLE
fix: Remove double click column edit option from shared form

### LINKED ISSUE #5197: 🐛 Bug: Activity.ListSharedView double click Table headers
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

open a link of Activity.ListSharedView,
double click any cell of table headers,
open a box of edit coloumn name...

### Desired Behavior

the box is a bug

### Project Details

Nocodb: v0.105.3
Database: mysql

### Attachments

nocodb-sharelink-tableheader

### LINKED ISSUE #5205: 🐛 Bug: In external form view, column definitions can be altered
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

1. Create a new table, define some columns
2. Create a new form view, and some of the columns
3. Copy the share link of the form view
4. Sign Out
5. Access the share link of the form view copied in step 3
6. double click on the description of any field and a dialog box opens where you can change column title and column type.

### Desired Behavior

It should not be possible to change column definitions of a table in a form that may be shared with untrusted third parties.

### Project Details

Node: **v16.19.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
RootDB: **sqlite3**
PackageVersion: **0.105.3**

I have also observed this behaviour on 0.105.2

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by pranavxc**
## Change Summary

- Hide the column edit option from the shared form
- Add logout API for clearing refresh token from cookie and database to avoid automatic token generation in GUI

closes #5205 
closes #5197

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by dstala — CHANGES_REQUESTED** [review]
works- verified for both form view & grid view share. 

minor change required, see if you can accommodate with ease
- on hover over column header, mouse icon changes to `click` icon, giving a false impression that its clickable. avoid changing mouse icon

File: packages/nocodb-sdk/src/lib/Api.ts [hunk 1]
```diff
@@ -1126,13 +1126,38 @@ export class Api<
...
+        {
+          msg?: string;
+        },
+        any
+      >({
+        path: `/api/v1/auth/user/signin`,
+        method: 'POST',
+        format: 'json',
+        ...params,
+      }),
```
Comment (wingkwong) [inline packages/nocodb-sdk/src/lib/Api.ts:1152]: name is `signout` but path is `signin`. please double check.

File: packages/nocodb-sdk/src/lib/Api.ts [hunk 2]
```diff
@@ -1126,13 +1126,38 @@ export class Api<
...
+
     /**
  * @description Authenticate existing user with their email and password. Successful login will return a JWT access-token. 
  * 
  * @tags Auth
  * @name Signin
  * @summary Signin
- * @request POST:/api/v1/auth/user/signin
+ * @request POST:/api/v1/auth/user/signout
```
Comment (wingkwong) [inline packages/nocodb-sdk/src/lib/Api.ts:1160]: name is `signout` but path is `signin`. please double check.

File: packages/nocodb-sdk/src/lib/Api.ts [hunk 3]
```diff
@@ -1151,7 +1176,7 @@ export class Api<
           msg?: string;
         }
       >({
-        path: `/api/v1/auth/user/signin`,
+        path: `/api/v1/auth/user/signout`,
```
Comment (wingkwong) [inline packages/nocodb-sdk/src/lib/Api.ts:1179]: name is `signout` but path is `signin`. please double check.

File: packages/nocodb/src/schema/swagger.json [hunk 4]
```diff
@@ -96,6 +96,34 @@
...
+          }
+        },
+        "tags": [
+          "Auth"
+        ],
+        "description": "Clear refresh token from the database and cookie."
+      },
+      "parameters": []
+    },
+    "/api/v1/auth/user/signout": {
```
Comment (wingkwong) [inline packages/nocodb/src/schema/swagger.json:126]: name is `signout` but path is `signin`. please double check.

**Review by dstala — COMMENTED** [review]
shared view edit menu bug is fixed. noticing issues with sign-in procedures.

**Review by dstala — APPROVED** [review]

**Pull request merged**